### ISSUE TITLE
Sidebar links not present / not working

### ISSUE THREAD

**Issue by jelmerveen** [vue-styleguidist/vue-styleguidist#960]
**Current behavior**

I run styleguidist on my Vue application, but the links in the menu do not work when I click on them.
They should navigate to /#section or /#componentname, but the href properties are missing

**To reproduce**
Install vue styleguidist in a  project
run
```
npm run styleguide
```

Go to localhost:6060 and try to navigate to a component.

Or checkout my repository
navigate to the folder library

```
npm install
```

```
npm run styleguide
```

Go to localhost:6060 and try to navigate to a component.

https://github.com/jelmerveen/vue-component-library-test/tree/visuals/styleguidist

**Expected behavior**
I expect the links in the sidebar to work.

**Comment by elevatebart** [comment]
Hello @jelmerveen 
Thank you for reporting this,

I will clone your repo and see if I can help.
Give me a few minutes.

**Comment by elevatebart** [comment]
In short, I do not find anything obviously wrong with you repo.

What browser are you using?

Here is what I tried 

```sh
git clone https://github.com/jelmerveen/vue-component-library-test.git
git checkout visuals/styleguidist
yarn install
yarn styleguide
```

I use yarn because of the yarn.lock file in the library folder. 
I can try to use npm instead.

**Comment by jelmerveen** [comment]
And the links are working? I tried firefox and chrome.

this is the html generated in the nav

```
<nav>
  <div class="rsg--search-51">
    <input
      class="rsg--input-52"
      placeholder="Filter by name"
      aria-label="Filter by name"
      value=""
    />
  </div>
  <ul class="rsg--list-53">
    <li class="rsg--item-54 rsg--isChild-55">
      <a class="rsg--link-48" data-testid="rsg-toc-link">app-button</a>
    </li>
    <li class="rsg--item-54 rsg--isChild-55">
      <a class="rsg--link-48" data-testid="rsg-toc-link">Counter</a>
    </li>
    <li class="rsg--item-54 rsg--isChild-55">
      <a class="rsg--link-48" data-testid="rsg-toc-link">TestCounter</a>
    </li>
  </ul>
</nav>

```

as you can see. no href="/#" present

**Comment by elevatebart** [comment]
I just tried with npm and I reproduce your issue. 
It could be due to an issue in a dependency.

I will check the package-lock

**Comment by jelmerveen** [comment]
thanks!

**Comment by elevatebart** [comment]
It seems to be because of a fix on react-styleguidist@11.0.9

As a workaround, you can run the following command, that will temporarily bring the links back

```
npm install react-styleguidist@11.0.8
```

I am working on a more definitive fix

**Comment by elevatebart** [comment]
You should be able to remove the react-styleguidist dependency
